STL_Extension/include/CGAL/Compact_container.h

```diff
@@ -31,6 +31,7 @@
 #include <CGAL/CC_safe_handle.h>
 #include <CGAL/Time_stamper.h>
 #include <CGAL/Has_member.h>
+#include <CGAL/Handle_hash_function.h>
```

Comment:
In case this include is really needed, and cannot be replace with a forward declaration (@lrineau can you tell us), then in several  files `Package/package_info/Package/dependencies` you must add `Hash_map`.  I copy paste from travis.

Differences in Triangulation: Hash_map are new and not committed.
Differences in Surface_mesh_simplification: Hash_map are new and not committed.
Differences in Surface_mesh_shortest_path: Hash_map are new and not committed. 
Differences in Surface_mesh_segmentation: Hash_map are new and not committed.
Differences in STL_Extension: Hash_map are new and not committed.
Differences in Spatial_searching: Hash_map are new and not committed.
Differences in Snap_rounding_2: Hash_map are new and not committed.
Differences in AABB_tree: Hash_map are new and not committed.

Reply:
From my experiments with gcc it looks like multiple forward declarations of a class template are not an issue so I will try that route.

Reply:
It can be a (forward-)declaration, without including the header that defines the template, because one can declare and define a partial specialization of a template that is only declared.